### PR TITLE
try to fix deploy:

### PR DESCRIPTION
**Pull request by hkollmann**
- move to github token instead of ssh
   - autoincrement version.txt during build

### CODE REVIEW

File: .travis/deploy [hunk 1]
```diff
@@ -1,37 +1,35 @@
...
 
+node -v
+npm -v
+git config user.email "travis@travis-ci.org"
+git config user.name "Travis CI"
+NEW_VERSION=$(npm version prerelease -m "[ci skip] new npm version: %s")
+echo "new version: $NEW_VERSION"
+echo $NEW_VERSION > version.txt
+GIT_URL="$(git config remote.origin.url)"
+GIT_URL=${GIT_URL/https:\/\//https:\/\/$GH_TOKEN@}
```
Comment (oetiker) [inline .travis/deploy]: what is the @ for ?

**Review by oetiker — APPROVED** [review]

**Comment by oetiker** [comment]
worth a try ...

**Review by level420 — APPROVED** [review]
Whatever makes it work again: do it!

**Review by cboulanger — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Comment by cajus** [comment]
I've no time to check it in the moment. I guess the easiest change would be to just call the travis cli tool to re-add the ssh key. If it does work now with the deploy key, and we're OK with the token stored unencrypted on the travis side (which makes only a very small difference to what we do in the moment), go for it!

**Review by cajus — APPROVED** [review]

**Review by derrell — APPROVED** [review]

**Comment by cboulanger** [comment]
merged, thank you @hkollmann and sorry @derrell, I was on mobile and didn't see that you were missing!

**Pull request merged**